tests/consistent-package-json.test.ts

```diff
@@ -139,6 +139,18 @@ packages.forEach(
           it('specifies the expected types', () => {
             expect(packageJSON.types).toBe(expectedPackageJSON.types);
           });
+
+          it('specifies the expected exports', () => {
+            expect(packageJSON.exports).toStrictEqual(
+              expect.objectContaining(expectedPackageJSON.exports),
+            );
+          });
+
+          it('specifies esnext as the first expected export', () => {
+            expect(packageJSON.exports['.'].esnext).toBe(
```

Comment:
This seems like it asserts the value of a specific key instead of anything to do with the ordering. It also seems to only check the `.` export key rather than all keys.

I would expect that something like the following would cause this test to fail.

```
exports: {
  "./foo" : { 'require': './foo.js', esnext: './foo.esnext' }
}
```